
---
float64 Hmin_verde
float64 Hmax_verde
float64 Vmin_verde
float64 Vmax_verde
float64 Smin_verde
float64 Smax_verde
float64 Hum_verde
float64 Hmin_negro
float64 Hmax_negro
float64 Vmin_negro
float64 Vmax_negro
float64 Smin_negro
float64 Smax_negro
float64 Hum_negro
float64 Hmin_rojo1
float64 Hmax_rojo1
float64 Vmin_rojo1
float64 Vmax_rojo1
float64 Smin_rojo1 
float64 Smax_rojo1
float64 Hum_rojo1
float64 Hmin_rojo2
float64 Hmax_rojo2
float64 Vmin_rojo2
float64 Vmax_rojo2
float64 Smin_rojo2
float64 Smax_rojo2
float64 Hum_rojo2
float64 act
int64 xmin1
int64 xmax1
int64 ymin1
int64 ymax1
int64 xmin2
int64 xmax2
int64 ymin2
int64 ymax2
float64 arprom_verde
float64 arprom_rojo
float64 arprom_negro
